doc/Articles/FAQ.md

```diff
@@ -133,6 +133,21 @@ NetworkServer.connections.Count
 - Socket connections, includes people without spawned prefabs, non authenticated, or that may have bugged out but during spawn, but have connected temporarily (Android users minimising game).  Only host / server can check this.
 
 NetworkManager.singleton.numPlayers
-- Number of active spawned player objects across all connections on the server.
-Only host / server can check this, below is an example of how to sync this number to all players.
+- Number of active spawned player objects across all connections on the server (only the host / server can check this)
```

Comment:
```suggestion
- Number of active spawned player objects on the server (only the host / server can check this)
```
There's only one player object per connection, so that phrase is redundant.